src/main/java/supersymmetry/mixins/gregtech/MultiblockMachineMixin.java

```diff
@@ -0,0 +1,31 @@
+package supersymmetry.mixins.gregtech;
+
+import gregtech.api.metatileentity.multiblock.MultiblockWithDisplayBase;
+import gregtech.common.metatileentities.multi.electric.MetaTileEntityFluidDrill;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.asm.mixin.Mixin;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+@Mixin(MetaTileEntityFluidDrill.class)
+public abstract class MultiblockMachineMixin extends MultiblockWithDisplayBase {
+
+    public MultiblockMachineMixin(ResourceLocation metaTileEntityId) {
```

Comment:
This should be named as MetaTileEntityFluidDrillMixin

Reply:
Do I have to make a seperate mixin if I target the ore drilling rig?

Reply:
yes if you want to make them rain-proof as well